shared/chat-sdk/reference/release-notes/unity.mdx

```diff
@@ -1,5 +1,68 @@
 <PlatformWrapper platform="unity">
 
+## v1.0.9
+
+v1.0.9 was released on February 1, 2023.
+
+#### New features
+
+1. Adds the following methods to the `SDKClient` class:     
+  - `GetLoggedInDevicesFromServer`: Gets the list of online devices on which you have logged in with a specified account.
+  - `KickDevice`: Logs out from a specified account on a device.
+  - `kickAllDevices`: Logs out from a specified account on all devices.
+2. Adds the following methods to the `RoomManager` class: 
+  - `FetchAllowListFromServer`: Gets the allow list of a chat room from the server.
+  - `CheckIfInRoomAllowList`: Checks whether the current member is on the chat room block list.
+  - `GetChatRoom`: Gets the details of a chat room from the memory.
+  - `UnMuteAllRoomMembers`: Unmutes all members of a chat room.
+3. Adds the following callbacks to the `IRoomManagerDelegate` class:
+  - `OnSpecificationChangedFromRoom`: Occurs when specifications of a chat room are changed.
+  - `OnAddAllowListMembersFromChatroom`: Occurs when a chat room member(s) is/are added to the allow list of a chat room.
+  - `OnRemoveAllowListMembersFromChatroom`: Occurs when a chat room member(s) is/are removed from the allow list of a chat room.
+  - `OnRemoveFromRoomByOffline`: Occurs when a member is removed from a chat room because he or she gets offline.             
+4. Adds the following callbacks to the `IConnectionDelegate` class:
+  - `OnLoggedOtherDevice`: Occurs when the user logs in to another device with the current account.
+  - `OnRemovedFromServer`：Occurs when the current user account is removed from the server.
+  - `OnForbidByServer`: Occurs when the current user account is banned by the server.
+  - `OnChangedIMPwd`：Occurs when the user is forced to log out of the current account because the login password is changed.
+  - `OnLoginTooManyDevice`: Occurs when the user is forced to log out of the current account because she or he has exceeded the maximum number of devices allowed for this account.
+  - `OnKickedByOtherDevice`：Occurs when the user is forced to log out of the current account from the current device due to the login to another device. 
+  - `OnAuthFailed`: Occurs when the user is forced to log out of the current account due to an authentication failure.
+5. Adds the following attributes to the `Group` class:           
+  - `IsMemberOnly`: Whether users can join a group only via a join request or a group invitation, but cannot join freely. 
+  - `IsMemberAllowToInvite`: Whether all the group members, other than the group owner and admins, can invite users to join the group.
+  - `MaxUserCount`：The maximum number of users that can join the group.
+  - `Ext`: The custom group extension information.
+  - `IsDisabled`: Whether the group is disabled.     
+              
+#### Improvements
+
```

Comment:
You used present tense for features (which is correct), but past tense here. A general recommendation is to stick to present.